docs/create_allocator_service.md

```diff
@@ -0,0 +1,435 @@
+
+Since the cert we created earlier is a self signed cert, ignoring insecure warning by adding the -k flag might be necessary:
+```
+curl -k https://the.ip.address/healthz
+```
+
+The output should look like this:
+```
+Healthy
+```
+
+You may need to wait a few moments longer if the output has ssl errors like this:
+```
+curl: (35) error:14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure
+```
+
+
+### 10. Check Game Servers
+Let's make sure that we have one or more Game Servers in a ready state by running this command:
+```
+kubectl get gs -o=custom-columns=NAME:.metadata.name,STATUS:.status.state,IP:.status.address,PORT:.status.ports
+```
+
+For a fleet of 2 replicas, you should see 2 Game Servers with a Status of Ready:
+```
+NAME                     STATUS    IP              PORT
+simple-udp-thbvz-mqntp   Ready     35.196.249.83   [map[name:default port:7374]]
+simple-udp-thbvz-smchl   Ready     35.229.96.214   [map[name:default port:7016]]
+```
+
+If there is no fleet, please review [Create Game Server Fleet](../docs/create_fleet.md).
+
+
+### 11. Allocate a Game Server
+The service uses Basic Auth to provide some security as to who can allocate gameserver resources.  A generated key is in main.go that you can change.
+
+Now that the ingress has been created and we know it is healthy, let's allocate a Game Server by passing in our user and key to the /address endpoint.
+
+Allocate a Game Server and retrieve its IP address and port by running this command:
+```
+curl -k -u v1GameClientKey:EAEC945C371B2EC361DE399C2F11E https://[the.ip.address]/address
+```
+
+The output should show an IP address and port, similar to this:
+```
+{"address":"35.229.96.214","port":7016}
+```
+
+Check the Game Servers again, and notice the Allocated Status.  You should see something like this:
+```
+NAME                     STATUS      IP              PORT
+simple-udp-thbvz-mqntp   Ready       35.196.249.83   [map[name:default port:7374]]
+simple-udp-thbvz-smchl   Allocated   35.229.96.214   [map[name:default port:7016]]
+```
+
+
+### 12. Make the IP Address Permanent
+Since the ephemeral IP address previously created will only live for as long as the service lives, allocating a permanent IP address is necessary for longer term reliability.
+
+:warning: This step may cause additional billing charges. :warning:
+
+Reserve a static external IP address named allocator-static-ip by running:
+```
+gcloud compute addresses create allocator-static-ip --global
+```
+
+Configure the existing Ingress to use the reserved IP address by adding an annotation to allocator-ingress.yaml:
+```
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "allocator-static-ip"
+```
+
+Apply the change by running:
+```
+kubectl apply -f allocator-ingress.yaml
+```
+
+
+### 13. Create an A Record
```

Comment:
As per your comments - let's not worry about this.

Reply:
Ok I will edit this out.